BitwardenShared/UI/Auth/StartRegistration/StartRegistrationView.swift

```diff
@@ -77,6 +65,30 @@ struct StartRegistrationView: View {
         .textFieldConfiguration(.email)
     }
 
+    /// The main content view that displays a scrollable layout of registration details.
+    private var mainContent: some View {
+        GeometryReader { proxy in
+            VStack(spacing: 0) {
+                if store.state.isCreateAccountFeatureFlagEnabled {
+                    Spacer(minLength: 24)
+
+                    Image(decorative: Asset.Images.logo)
+                        .resizable()
+                        .scaledToFit()
+                        .foregroundColor(Asset.Colors.iconSecondary.swiftUIColor)
+                        .frame(maxWidth: .infinity, maxHeight: 34)
+                        .padding(.horizontal, 12)
+
+                    Spacer(minLength: 24)
+                }
+
+                registrationDetails
+            }
+            .frame(minHeight: store.state.isCreateAccountFeatureFlagEnabled ? proxy.size.height : 0)
+            .scrollView(showsIndicators: false)
```

Comment:
⛏️ I think if we fix the frame to the height of the view, we need to remove the padding that `scrollView` adds and manually add it before the frame modifier. Otherwise, the content size is larger than the scroll bounds and it will scroll slightly.

Reply:
Did i fix this right?  my eyes are being deceiving right now... the min spacer height i think is messing with me.  Did i add a double top padding here?

Reply:
I added a conditional for the top padding and it looks better to me